tests/test_parallel_upload_download.cpp

```diff
@@ -115,6 +115,18 @@ void test_parallel_upload_core(int fileNumber)
   }
   assert_int_equal(status, SF_STATUS_EOF);
 
+  std::string getCommand = "get @%test_parallel_upload_download file://" + dataDir + "test_parallel_download";
+  result = agent.execute(&getCommand);
+  assert_int_equal(fileNumber, result->getResultSize());
+
+  while(result->next())
+  {
+    assert_string_equal("SUCCEED", result->getStatus());
+  }
+
+  std::string compCmd = "diff " + dataDir + "test_parallel_upload/ " + dataDir + "test_parallel_download/";
```

Comment:
diff won't work on windows.

Reply:
I understand that. Maybe need better ways to handle this once moving to windows. Either I need to do line by line checking or calculate the hash of files. Or some native windows system call to diff on two directories.